IyiKuyoro/TalesByYomi
# #161778534: setup the REST API for the project

### What does this PR do?
The PR sets up the REST API for the project using express

### Tasks done
1. Install all dependencies and developement dependecies
2. Setup transpiling with bable for production and development
3. Setup the app
4. Setup the default routes
5. Setup the default controller

### How can this be manually tested
1. Clone this repo
2. Checkout `ch-setup-REST-API-#161778534`
3. Run `npm i`
4. Run `npm run start`
    > - A `dist` folder should be created in the project directory with the transpiled code, and the app should be served and running on port 3000.
    > - On postman, test this routes `localhost:3000/`.
5. You can also start the server in developement mode with `npm run start_dev`

## Relevant PT stories
#161778534
